packages/widgets/package.json

```diff
@@ -20,17 +20,16 @@
     "peerDependencies": {
         "react": "^16.8",
         "react-dom": "^16.8",
-        "styled-jsx": "^3.2"
+        "styled-jsx": "^3.2",
+        "@dhis2/ui-core": "5.0.3",
+        "@dhis2/ui-icons": "5.0.3",
+        "@dhis2/ui-constants": "5.0.3",
+        "@dhis2/app-runtime": "^2.2.1",
+        "@dhis2/d2-i18n": "^1"
     },
     "dependencies": {
-        "@dhis2/app-runtime": "^2.2.1",
-        "@dhis2/d2-i18n": "^1",
         "@dhis2/prop-types": "^1.6.4",
-        "@dhis2/ui-constants": "5.0.3",
-        "@dhis2/ui-core": "5.0.3",
-        "@dhis2/ui-icons": "5.0.3",
-        "classnames": "^2.2.6",
-        "resize-observer-polyfill": "^1.5.1"
```

Comment:
Why was `resize-observer-polyfill` removed?

Reply:
@martinkrulltott it's not used in `ui-widgets`, only in `ui-core` (which still has the dependency).  I think we might want to remove the dep from `ui-core` as well at some point, since typically it should be the application's responsibility to polyfill missing features for a given browser, but that's a separate issue